includes/class-amp-theme-support.php

```diff
@@ -229,7 +229,7 @@ public static function add_hooks() {
 		 * Disable admin bar because admin-bar.css (28K) and Dashicons (48K) alone
 		 * combine to surpass the 50K limit imposed for the amp-custom style.
 		 */
-		add_filter( 'show_admin_bar', '__return_false', 100 );
+		add_filter( 'show_admin_bar', '__return_false', 100 ); // phpcs:ignore
```

Comment:
This actually isn't necessary. The PHPCS ruleset does not include the [AdminBarRemoval](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress/Sniffs/VIP/AdminBarRemovalSniff.php) WordPress.com VIP sniff. You should make sure to use the project's own [ruleset](https://github.com/Automattic/amp-wp/blob/develop/phpcs.xml) when developing.